proxy/make.js

```diff
@@ -112,6 +112,30 @@ task('node', 'build the node binary', function() {
   console.log('Node prepared.');
 });
 
+task('pack-webext', 'pack the webextension for deployment', function() {
+  try {
+    execSync(`rm -f source.zip`);
+    execSync(`rm -f webext/webext.zip`);
+  } catch (error) {
+    console.log('Error removing zip files');
+  }
+  execSync(`git submodule update --remote`);
+  var version = process.argv[3];
+  console.log(version);
+  var manifest = require('./webext/manifest.json')
+  manifest.version = version;
+  writeFileSync('./webext/manifest.json', JSON.stringify(manifest, null, 2), 'utf8');
```

Comment:
I think I'm missing something here but manifest.json contains the line `"description": "__MSG_appDesc__",`. Isn't this supposed to be substituted by a (translated) string? When I run `node make.js pack-webext foo`, the resulting webext.zip contains the original manifest.json, including `__MSG_appDesc__`.

Reply:
The description is a localizable property of the manifest. See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Internationalizing_manifest.json